.github/workflows/parsely.yml

```diff
@@ -22,13 +22,28 @@ jobs:
       fail-fast: false
       matrix:
         config:
+          # Oldest version of the parsely plugin
           - { wp: latest, parsely: '3.1', mode: 'filter_enabled', php: '7.4' }
+          - { wp: latest, parsely: '3.1', mode: 'filter_disabled', php: '7.4' }
```

Comment:
- Added `filter_disabled`, `option_disabled`, `filter_and_option_disabled` modes.
- Previously we were using some random parsely versions for testing but now in this PR we updated `parsely` plugin version to use the oldest version, some mid version and latest available version (without pinning i.e. removed parsely key from config object)

Reply:
This feels like a lot of test runs, not sure how beneficial all these cases are going to be vs how long they're going to be holding up every PR :/

- Are the `filter_and_option_enabled` & `filter_and_option_disabled` cases really needed for example?
- I think we should just do 1 version of parsely (latest)

Reply:
Removed the tests of 1 version because I think we should have at least one oldest version also which should flag issues when we are expecting something to happen but it can't due to lack of filter.